WORKSPACE

```diff
@@ -43,10 +53,6 @@ kotlin_repositories()
 load("@io_bazel_rules_kotlin//kotlin:core.bzl", "kt_register_toolchains")
 kt_register_toolchains()
 
-# Load //builder/python
-load("@vaticle_dependencies//builder/python:deps.bzl", python_deps = "deps")
-python_deps()
```

Comment:
define python earlier to ensure we get the most recent version of `rules_python`, which lots of our bazel dependencies may load their own versions of if they need to (using an old version!)